contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -253,10 +253,19 @@ including configuration settings provided by the user,
 or otherwise changed after creation by other ecosystem components (e.g.,
 schedulers, auto-scalers), and is persisted in stable storage with the API
 object. If the specification is deleted, the object will be purged from the
-system. The status summarizes the current state of the object in the system, and
+system.
+
+The status summarizes the current state of the object in the system, and
 is usually persisted with the object by automated processes but may be
-generated on the fly. At some cost and perhaps some temporary degradation in
-behavior, the status could be reconstructed by observation if it were lost.
+generated on the fly.  As a rule of thumb, fields in status are the most recent
```

Comment:
Small suggestion here:
```suggestion
generated on the fly.  As a general guideline, fields in status are the most recent
```
Because:
- idioms are less clear for non-native speakers
- this particular one has a contentious history of being perceived as problematic.

Because the history is contentious, I don't know if it's actually a problem, but easier to just skip the whole thing and use a clearer, non-idiomatic term.

Reply:
I did not know the history.  Thanks.  People suck. :)  Will change it.

Reply:
Damn! thanks for pointing this out @youngnick